lib/console/deployments/pr/impl/bitbucket.ex

```diff
@@ -50,20 +50,97 @@ defmodule Console.Deployments.Pr.Impl.BitBucket do
 
   def review(conn, %PullRequest{url: url}, body) do
     with {:ok, group, number} <- get_pull_id(url),
-         {:ok, conn} <- connection(conn) do
-      case post(conn, Path.join(["/repositories", "#{URI.encode(group)}", "pullrequests", number, "comments"]), %{
-        content: %{
-          raw: filter_ansi(body),
-          markup: "markdown"
+         {:ok, conn} <- connection(conn),
+         {:ok, %{"id" => id}} <- post(conn, Path.join(["/repositories", "#{URI.encode(group)}", "pullrequests", number, "comments"]), %{
+           content: %{
+             raw: filter_ansi(body),
+             markup: "markdown"
+           }
+         }) do
+      {:ok, "#{id}"}
+    else
+      {:error, error} -> {:error, error}
+      _ -> {:error, "unknown bitbucket error"}
+    end
+  end
+
+  def files(conn, url) do
+    with {:ok, group, number} <- get_pull_id(url),
+         {:ok, conn} <- connection(conn),
+         {:ok, %HTTPoison.Response{status_code: 200, body: body}} <- HTTPoison.get("#{conn.host}/repositories/#{URI.encode(group)}/pullrequests/#{number}", Connection.headers(conn)),
```

Comment:
move the URI.encode to the top and save to a var so it doesn't have to be repeated